### PR TITLE
Adding detail on NEMO physical models

### PR DESCRIPTION
**Pull request by jtneedels**
Adding discussion of NEMO physical models for thermo, chemistry, vibe relaxation, transport models, etc.

### CODE REVIEW

**Review by pcarruscag — APPROVED** [review]

File: _docs_v7/Thermochemical-Nonequilibrium.md [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+
+where $\sigma_s$ is the effective collision~cross-section.
+
+---
+
+# Viscous Phenomena and Transport Coefficients #
+
+| Solver | Version | 
+| --- | --- |
+| `NEMO_EULER`, `NEMO_NAVIER_STOKES` | 7.0.0 |
```
Comment (WallyMaier) [inline _docs_v7/Thermochemical-Nonequilibrium.md:146]: Would this only be navier-stokes?

File: _docs_v7/Thermochemical-Nonequilibrium.md [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+
+$$
+D_{e,r} = \frac{k_{B} T_{ve}}{p \Delta_{e,r}^{(1)}(T_{ve})}.
+$$
+
+## Sutherland Viscosity Model ##
+
+In addition to the two models discussed above, there is the option to use a Sutherland model to calculate the flow viscosity. The Sutherland model is not applicable at high temperatures.
+
+In this case the viscosity is computed as
```
Comment (WallyMaier) [inline _docs_v7/Thermochemical-Nonequilibrium.md:331]:
```suggestion
In this case, the viscosity is computed as
```

File: _docs_v7/Thermochemical-Nonequilibrium.md [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+
+If the Sutherland model is selected with a NEMO solver, species diffusion coefficients and thermal conductivity are computed using the models described in the Wilkes-Blottner-Eucken section.
+
+---
+
+# Slip Flow #
+
+| Solver | Version | 
+| --- | --- |
+| `NEMO_EULER`, `NEMO_NAVIER_STOKES` | 7.0.0 |
```
Comment (WallyMaier) [inline _docs_v7/Thermochemical-Nonequilibrium.md:347]: same here

File: _docs_v7/Thermochemical-Nonequilibrium.md [hunk 4]
```diff
@@ -0,0 +1,398 @@
...
+
+SU2-NEMO uses the Maxwell velocity and Smoluchowski temperature jump  equations to compute the velocity and temperature of the gas in contact with the surface. The equations are given as
```
Comment (WallyMaier) [inline _docs_v7/Thermochemical-Nonequilibrium.md:349]:
```suggestion
SU2-NEMO uses the Maxwell velocity and Smoluchowski temperature jump equations to compute the velocity and temperature of the gas in contact with the surface. The equations are given as
```

File: _docs_v7/Thermochemical-Nonequilibrium.md (hunk 1 above)
Comment (jtneedels) [reply _docs_v7/Thermochemical-Nonequilibrium.md:146]: Yup good catch, I'll correct it.

File: _docs_v7/Physical-Definition.md [hunk 5]
```diff
@@ -104,6 +105,18 @@ A chemistry model, consisting of a set of flow species, thermochemical propertie
 
 Free-stream mass fractions must also be specified in list using the option `GAS_COMPOSITION`. The mass fractions are specified as decimal values in the order of the species in the gas model. For example, an AIR-5 mixture of 77% oxygen and 23% nitrogen would be expressed as (0.77, 0.23, 0.00, 0.00, 0.00).
 
+## Transport Coefficient Model (Thermochemical Nonequilibrium) ##
+
+| Solver | Version |
+| --- | --- |
+| `NEMO_NAVIER_STOKES` | 7.0.0 |
+
+A transport coefficient model must be specified for viscous simulations with the NEMO solver, using the `TRANSPORT_COEFF_MODEL` config option. Available options are the Wilkes-Blottner-Eucken, Gupta-Yos, and Sutherland viscosity models, specified by `WILKE`, `GUPTA-YOS`, and `SUTHERLAND`, respectively. The default option for transport coefficient model is Wilkes-Blottner-Eucken.
```
Comment (WallyMaier) [inline _docs_v7/Physical-Definition.md:114]: We have Chapman-Enskog for mutation as well.

**Pull request merged**